docs/types.rst

```diff
@@ -155,6 +155,10 @@ API will build them for you, as the result of various API methods.
 
 Each ``Transaction`` has the following attributes:
 
+-  ``legacy tag``: A short message attached to the transaction.
```

Comment:
s/b `legacy_tag`

Also, could we include a little information about the difference between `legacy_tag` and `tag`?